benchmark/runner/tachi.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/app/build.gradle.kts b/app/build.gradle.kts
```

Comment:
Maybe you should create a separate directory for each benchmark containing the code needed to generate the dependency jars? This patch file is in the runner dir which seems a little arbitrary. :)